src/plugins/discourse/mirrorRepository.js

```diff
@@ -61,6 +62,11 @@ export interface ReadRepository {
   ): $ReadOnlyArray<TopicId>;
 }
 
+export type SyncHeads = {|
+  +definitionCheckMs: number,
```

Comment:
it would be helpful to have a comment explaining what definitionCheckMs signifies. (i assume it's the timestamp when we last did an explicit search for category definition topics, since those aren't returned like normal topics--but the next person reading this code may not have that context fresh in their minds)

Reply:
I now realize this information is in the commit message. :) Would be nice to copy the same info here.

Reply:
Added descriptions to the type comments.